### ISSUE TITLE
Error running yolov5 model

### ISSUE THREAD

**Issue by RajbharSachin** [spyder-ide/spyder#21832]
### What steps will reproduce the problem?

Follow the vedio till 7:30 then run, it will reproduce the error: https://www.youtube.com/watch?v=msGOrJIMGy8&ab_channel=NicholasRenotte

I was working to build a ML Drowsiness Detector

### Traceback
```python-traceback
Exception in comms call get_namespace_view:
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 343, in _handle_remote_call
    return_value = self._remote_callback(
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\frontendcomm.py", line 267, in _remote_callback
    return super(FrontendComm, self)._remote_callback(
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 356, in _remote_callback
    return self._remote_call_handlers[call_name](
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\console\kernel.py", line 189, in get_namespace_view
    view = make_remote_view(ns, settings, EXCLUDED_NAMES)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\utils\nsview.py", line 677, in make_remote_view
    'type':  get_human_readable_type(value),
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\utils\nsview.py", line 528, in get_human_readable_type
    elif isinstance(item, PIL.Image.Image):
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\utils\lazymodules.py", line 55, in __getattr__
    return getattr(self.__spy_mod__, name)
AttributeError: module 'PIL' has no attribute 'Image'
```

## Versions

* Spyder version: 5.2.2 
* Python version: 3.9.13
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0;<8.0.0        :  7.31.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.16.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.4.0 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.16.1 (OK)
pylint >=2.5.0                :  2.14.5 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.0.0 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.2.0 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  69.0.2 (OK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  0.29.32 (OK)
matplotlib >=2.0.0            :  3.5.2 (OK)
numpy >=1.7                   :  1.26.0 (OK)
pandas >=1.1.1                :  1.4.4 (OK)
scipy >=0.17.0                :  1.11.3 (OK)
sympy >=0.7.3                 :  1.10.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @RajbharSachin, thanks for reporting. This problem should be fixed in our latest version available in Anaconda (5.4.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

Let us know if it works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
